### PR TITLE
Add boolean to handler to know if they are Standardized or not

### PR DESCRIPTION
**Pull request by jaumard**
For now test is done for `trails-controller` and `trails-policy` but it have to be replace by to standardized classes when they will be created. 

Also update dependencies

Fix https://github.com/trailsjs/trailpack-router/issues/47

### CODE REVIEW

File: lib/schemas/route.js [hunk 1]
```diff
@@ -35,6 +36,16 @@ module.exports = joi.object().keys({
...
+      }),
+      //Express definition
+      joi.object().keys({
+        origin: joi.any(),
+        methods: [joi.string(), joi.array()],
+        allowedHeaders: [joi.string(), joi.array()],
+        exposedHeaders: [joi.string(), joi.array()],
+        maxAge: joi.number(),
+        preflightContinue: joi.boolean(),
+        credentials: joi.boolean()
```
Comment (jaumard) [inline lib/schemas/route.js]: @tjwebb can I have your feedback about this please ?

File: lib/util.js [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 const _ = require('lodash')
 const call = require('call')
+const Controller = require('trails-controller')
+const Policy = require('trails-policy')
 const lib = require('.')
 
+const isClassStandardized = {} // cache to know if class is standar or not
```
Comment (tjwebb) [inline lib/util.js]: This all looks good except I don't like storing state in a module-global variable. Also, indexing by `className` isn't reliable. The whole `api.x.y.z` namespace needs to be used.

Comment (jaumard) [inline lib/util.js]: @tjwebb I remove totally the cache and instead adding a boolean to the class itself.
But in order to this to work we need to create a `trails-standardized-controller` and `trails-standardized-policy`, like this the current one is used for specific webserver api and this ones for unified api. Can I create them ?

**Comment by jaumard** [comment]
With https://github.com/trailsjs/trailpack-policy/blob/master/index.js#L11 and https://github.com/trailsjs/trailpack-controller/blob/master/index.js#L11

Are you ok with this @tjwebb @konstantinzolotarev ? If yes I'll close this revert as it's not needed anymore.

**Comment by konstantinzolotarev** [comment]
@jaumard I'm ok with it. @tjwebb any comments ?

**Comment by jaumard** [comment]
@trailsjs/maintainers up :)

**Pull request closed without merging**